src/test/lib/log-handler-test.js

```diff
@@ -0,0 +1,203 @@
+                assert.equal(loggerTesting.transports['log-file'].level, 'info');
+                done();
+            });
+
+            it('Should return a winston logger object with level "info"', function (done) {
+                const loggerTesting = logger.createInstance('log-file', 'info');
+                assert.equal(loggerTesting.transports.console.level, 'info');
+                assert.equal(loggerTesting.transports['log-file'].level, 'info');
+                done();
+            });
+
+            it('Should return a winston logger object with level "warn"', function (done) {
+                const loggerTesting = logger.createInstance('log-file', 'warn');
+                assert.equal(loggerTesting.transports.console.level, 'warn');
+                assert.equal(loggerTesting.transports['log-file'].level, 'warn');
+                done();
+            });
+
+            it('Should return a winston logger object with level "error"', function (done) {
+                const loggerTesting = logger.createInstance('log-file', 'error');
+                assert.equal(loggerTesting.transports.console.level, 'error');
+                assert.equal(loggerTesting.transports['log-file'].level, 'error');
+                done();
+            });
+
+            it('Should return a error for winston logger object with unknown level', function (done) {
+                assert.throws(function (xdone) {
+                    logger.createInstance('log-file', 'banana');
+                    xdone();
+                });                          
+                done();
+            });
+
+
+            it('Should create the proper path if given', function (done) {
+                const loggerTesting = logger.createInstance('log-file', 'error','Sub-test');
+                assert.ok(fs.lstatSync(path.join(__dirname,"../../../logs/Sub-test")).isDirectory());
+                done();
+            });
+
+            it('Should write to console by default & colorize', function (done) {
+              const loggerTesting = logger.createInstance('log-file');
+             assert.equal(loggerTesting.transports.console.colorize, true);
+                 done();
+             });
+
+
+             it('Should not write to console if consoleFlage=false', function (done) {
+                const loggerTesting = logger.createInstance('log-file', 'error', '',false);
+                assert.throws(function (xdone) {
+                    loggerTesting.transports.console;
+                    xdone();
+                });           
+                 done();
+                });
+
+        });
+
+        describe('createGroupInstance.', function () {
+
+                it('Should write to console by default & colorize', function (done) {
+                const loggerTesting = logger.createGroupInstance('log-file');
+               assert.equal(loggerTesting.transports.console.colorize, true);
```

Comment:
Please indent the `assert` line

```suggestion
                 assert.equal(loggerTesting.transports.console.colorize, true);
```